src/components/toast/toast.scss

```diff
@@ -0,0 +1,100 @@
+@import "../../styles/freshworks.scss";
+
+:host {
+  display: none;
```

Comment:
Can you add font family here?

Reply:
it's added in freshworks.scss file that imported above